app/src/main/java/edu/grinnell/grinnell_publications_android/Models/Realm/RealmStory.java

```diff
@@ -0,0 +1,96 @@
+package edu.grinnell.grinnell_publications_android.Models.Realm;
+
+import java.util.List;
+
+import edu.grinnell.grinnell_publications_android.Models.Interfaces.Author;
+import edu.grinnell.grinnell_publications_android.Models.Interfaces.Publication;
+import edu.grinnell.grinnell_publications_android.Models.Interfaces.Reactions;
+import edu.grinnell.grinnell_publications_android.Models.Interfaces.Story;
+import io.realm.Realm;
+import io.realm.RealmList;
+import io.realm.RealmObject;
+
+/**
+ * This class implements @code{Story} using Realm Persistence
+ * @author Albert Owusu-Asare
+ * @since 1.1 Fri May  6 10:01:38 CDT 2016
+ * @see Story
+ * @see RealmObject
+ */
+public class RealmStory extends RealmObject implements Story{
```

Comment:
I think we recently added a new field to our article model -- "LastUpdatedAt". More discussion on that, and why it is used in the API doc (lt's already been resolved, so look under the comments tab).